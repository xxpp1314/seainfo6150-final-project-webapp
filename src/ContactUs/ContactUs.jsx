import React from 'react';
import styles from "./ContactUs.module.css";
import ContactUs from "../images/ContactUs.jpg";

const ContactUS = () => {
    return (            
        <div className={styles.mainpage}>
            <div className={styles.grid}>
                    <div className={styles.flex}>
                        <div className={styles.flexitems}>
                            <img className={styles.image} src={ContactUs} alt=""></img>
                        </div>
                        <div className={styles.gap}> </div>
                        <div className={styles.flexitems}>
                            <h1>Contact Information - Help You Find A Suitable Pet</h1><br /><br />
                            <h2>Address</h2>
                            <p>1234 Pet Road, Seattle, WA, 98000</p>
                            <h2>Call At</h2>
                            <p><a href="tel:+11234567890">+1 123-456-7890</a></p>
                            <h2>Fax</h2>
                            <p><a href="fax:+17894561230">+1 987-654-3210</a></p>
                            <h2>Email</h2>
                            <p><a href="mailto:pethome@gmail.com">pethome@gmail.com</a></p>
                        </div>
                    </div>
            </div>   
        </div>
    )
}

export default ContactUS