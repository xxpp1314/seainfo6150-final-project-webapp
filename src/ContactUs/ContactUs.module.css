  .mainpage{ 
    background-image: linear-gradient(to right,rgb(83, 15, 15), rgb(49, 12, 25) 70%);
    align-items: center;
    justify-content: center;
    padding: 5%;
    font-family: 'Arima Madurai', cursive;
    font-size: medium;
    color: rgb(221, 193, 193);
}
.grid {
    display: grid;
    grid-template-rows: 1fr;
    box-shadow: 0px 0px 5px 2px rgba(204, 0, 0, 0.5);    
    border: 0px solid grey;
    border-radius: 1%;
    justify-content: center;
    align-items: center;
}

.flex{
  padding: 5%;
  gap: 5%;
  display: flex;
  flex-direction: row;
  align-items: center;
}

@media screen and (min-width: 100px) {
  .flex {
    flex-direction: column;
  }
}

@media screen and (min-width: 400px) {
  .flex {
    flex-direction: column;
  }
}

@media screen and (min-width: 800px) {
  .flex {
    flex-direction: row;
    align-items: center;
    justify-items: center;
  }
}

.text{
  color: indianred;
}

.flexitems{
  align-items: center;
  justify-items: center;
}

.image{
  height: 100%;
  width: 100%;
  box-shadow: 0px 0px 5px 2px rgba(204, 0, 0, 0.5);    
  border: 0px solid grey;
  border-radius: 50%;
}

@media screen and (min-width: 100px) {
  .image {
    height: 50%;
    width: 100%;
  }
}

@media screen and (min-width: 300px) {
  .image {
    height: 60%;
    width: 100%;
  }
}

@media screen and (min-width: 500px) {
  .image {
    height: 50%;
    width: 100%;
  }
}

@media screen and (min-width: 800px) {
  .image {
    height: 80%;
    width: 100%;
  }
}