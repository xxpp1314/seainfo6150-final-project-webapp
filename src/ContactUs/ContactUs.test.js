import React from "react";
import { render } from "@testing-library/react";
import ContactUs from "./ContactUs.jsx";

it("renders the ContactUs Page correctly", () => {
  const { container } = render( 
    <ContactUs/>
  );
  expect(container).toMatchSnapshot();
});