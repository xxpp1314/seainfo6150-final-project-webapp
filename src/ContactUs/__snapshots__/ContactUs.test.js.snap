// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ContactUs Page correctly 1`] = `
<div>
  <div
    class="mainpage"
  >
    <div
      class="grid"
    >
      <div
        class="flex"
      >
        <div
          class="flexitems"
        >
          <img
            alt=""
            class="image"
            src="ContactUs.jpg"
          />
        </div>
        <div
          class="gap"
        >
           
        </div>
        <div
          class="flexitems"
        >
          <h1>
            Contact Information - Help You Find A Suitable Pet
          </h1>
          <br />
          <br />
          <h2>
            Address
          </h2>
          <p>
            1234 Pet Road, Seattle, WA, 98000
          </p>
          <h2>
            Call At
          </h2>
          <p>
            <a
              href="tel:+11234567890"
            >
              +1 123-456-7890
            </a>
          </p>
          <h2>
            Fax
          </h2>
          <p>
            <a
              href="fax:+17894561230"
            >
              +1 987-654-3210
            </a>
          </p>
          <h2>
            Email
          </h2>
          <p>
            <a
              href="mailto:pethome@gmail.com"
            >
              pethome@gmail.com
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
